inference-engine/src/readers/ir_reader/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_ENGINE_PLU
 target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
 
 target_link_libraries(${TARGET_NAME} PRIVATE ngraph
+                                             ir_frontend
+                                             frontend_manager
```

Comment:
we should not use `frontend_manager` dynamic library in IE libraries. Can we use `ir_frontend` directly?

Reply:
secondly, I dislike the fact that IR reader is not removed, but is split into 2 parts. Maybe we need to have a single library, but with 2 interfaces like for ONNX FE (ONNX FE + ONNX editor capability). For IR FE we can have 2 capabilities: IR FE + IR reader

Reply:
IR reader will be removed and it is currently in progress.

Reply:
IR reader was removed in https://github.com/openvinotoolkit/openvino/pull/7249